tests/bsim/bluetooth/host/misc/sample_test/prj.conf

```diff
@@ -0,0 +1,55 @@
+# Select only the config options that are necessary for the test.
```

Comment:
Maybe add something like: "Do not copy these comments into your tests' prj.conf"